study.md

```diff
@@ -50,15 +50,15 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+<!-- Responsible to handle CRUD  -->
```

Comment:
The controller is responsible for communication between the router and the model layer, and rendering responses to the client.